docs/content/config/index.md

```diff
@@ -0,0 +1,79 @@
+HedgeDoc can be configured via environment variables either directly or via an `.env` file.
+
+## The `.env` file
+
+The `.env` file should be places in the root of the project and contains key value pairs of environment variables, and the values you configured them with. This can for example look like this:
```

Comment:
```suggestion
The `.env` file should be placed in the root of the project and contains key-value pairs of environment variables, and the corresponding value. This can for example look like this:
```